package.json

```diff
@@ -35,13 +35,6 @@
     "@nestjs/common": "^8.2.2",
     "@nestjs/core": "^8.2.2",
     "@nestjs/microservices": "^8.2.2",
-    "@semantic-release/changelog": "^5.0.1",
```

Comment:
These should stay.

Reply:
I got errors on these, there are conflicting dependencies, with "semantic-release"